TYPO3.Media/Classes/TYPO3/Media/Domain/Model/Tag.php

```diff
@@ -37,6 +37,10 @@ class Tag
      */
     protected $assetCollections;
 
+    /**
+     * @param string $label
+     * @return void
```

Comment:
hate to break it to you, but `__construct` doesn't get the privilege of a `@return` annotation